### PR TITLE
setup routes and pages

### PR DESCRIPTION
**Pull request by randomicons**

### CODE REVIEW

**Review by arshtsingh — APPROVED** [review]

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Route, Routes } from "react-router-dom";
+import { Home } from "./pages/Home";
+import Dashboard from "./pages/Dashboard";
+import Search from "./pages/Search";
+import IssueDetails from "./pages/IssueDetails";
+
+export default function App() {
+  return (
+    <div>
```
Comment (enricozb) [inline frontend/src/App.tsx:9]: Why is this surrounded in a div?

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -2,12 +2,15 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "@fortawesome/fontawesome-free/css/all.css";
 
-import { Home } from "./pages/Home";
+import { BrowserRouter as Router } from "react-router-dom";
```
Comment (enricozb) [inline frontend/src/index.tsx:5]: group imports. external imports should be grouped together

File: frontend/src/pages/Home.tsx [hunk 3]
```diff
@@ -5,13 +5,13 @@ type BlockProps = {
...
+  <div className="rounded-2xl shadow-xl hover:shadow-2xl flex h-1/2 w-1/2 flex items-center justify-center bg-slate-800 text-white font-bold">
     {props.children}
   </div>
 );
 
 export const Home = () => (
-  <div className="absolute flex h-full w-full items-center justify-center gap-10 bg-gray-400">
-    <Block>{"hello, friend."}</Block>
+  <div className="absolute flex h-full w-full items-center justify-center gap-10 bg-slate-800">
+    <Block>{"hello, 🫃."}</Block>
```
Comment (enricozb) [inline frontend/src/pages/Home.tsx:15]: why?

File: frontend/src/index.tsx (hunk 2 above)
Comment (randomicons) [reply frontend/src/index.tsx:5]: No

File: frontend/src/App.tsx (hunk 1 above)
Comment (randomicons) [reply frontend/src/App.tsx:9]: Yeah I'll get rid of it

File: frontend/src/pages/Home.tsx (hunk 3 above)
Comment (randomicons) [reply frontend/src/pages/Home.tsx:15]: Must be a github glitch

**Pull request merged**